browser/scripts/checkbrowser.js

```diff
@@ -0,0 +1,45 @@
+checkBrowser = function() {
+	var agent = navigator.userAgent;
+	if ((/Chrome/i.test(agent)) || (/Firefox/i.test(agent))) {
+		// Good boy browser
+	}   
+    else if (/Android|webOS|CriOS|iPhone|iPad|iPod|BlackBerry|IEMobile|Opera Mini|Mobile|mobile/i.test(agent)) {
```

Comment:
well, as long as util.js loaded vefore application.js, I think whole thing can be moved there.